modules/config-idp.adoc

```diff
@@ -6,8 +6,11 @@
 [id="config-idp_{context}"]
 = Configuring an identity provider
 
+After you have installed {product-title}, you must configure your cluster to use an identity provider. You can then add members to your identity provider to grant them access to your cluster.
 
-After your {product-title} cluster is created, you must configure identity providers to determine how users log in to access the cluster. This example configures a GitHub identity provider.
+You can configure different identity provider types for your {product-title} cluster. Supported types include GitHub, GitHub Enterprise, GitLab, Google, LDAP, and OpenID Connect identity providers.
```

Comment:
Is `GitHub Enterprise` `HTPasswd` ? I didn't find `GitHub Enterprise` but `HTPasswd` . Better to match the UI. @yuwang-RH Would you please double confirm?

Reply:
Thank you for highlighting this. I was going on what is [listed in the OSD product docs](https://docs.openshift.com/dedicated/identity_providers/config-identity-providers.html#understanding-idp-supported_config-identity-providers) currently as a supported IDP. I left out HTPasswd because of [this Jira request](https://issues.redhat.com/browse/OSDOCS-2943).

Based on my comment this morning in that Jira and what is in the UI, I will add HTPasswd back in. I will also keep GitHub Enterprise in, because it is listed as supported in the docs. Looking at the corresponding procedure section, you can select **GitHub** in the drop-down menu to configure either a GitHub or GitHub Enterprise IDP.